src/transformers/training_args.py

```diff
@@ -378,6 +382,13 @@ class TrainingArguments:
             "help": "When resuming training, whether or not to skip the first epochs and batches to get to the same training data."
         },
     )
+    fp16_backend: str = field(
+        default="auto",
+        metadata={
+            "help": "The backend to used for mixed precision. Should be one of 'auto', 'amp' or 'apex'.",
+            "choices": ["auto", "amp", "apex"],
```

Comment:
Do we need to repeat the options twice? The options already list the possible options:
```
  --fp16_backend {auto,amp,apex}
                        The backend to used for mixed precision. Should be one
                        of 'auto', 'amp' or 'apex'.
                        
```

Reply:
oh, and a small typo - "backend to be used" or "backend to use" :)

Reply:
I removed the choices as you suggested.